test/bases/renku_data_services/data_api/test_users.py

```diff
@@ -205,3 +205,92 @@ async def test_logged_in_user_check_adds_user_if_missing(sanic_client, users, ad
         for iuser in res.json
     ]
     assert user in users_response
+
+
+@pytest.mark.asyncio
+async def test_delete_user(sanic_client, user_headers, app_config, users) -> None:
+    # Create an admin user
+    admin = UserInfo(
+        id="admin-id",
+        first_name="Admin",
+        last_name="Adminson",
+        email="admin@gmail.com",
+        namespace=Namespace(
+            id=ULID(),
+            slug="admin.adminson",
+            kind=NamespaceKind.user,
+            underlying_resource_id="admin-id",
+            created_by="admin-id",
+        ),
+    )
+    admin_token = {
+        "id": admin.id,
+        "is_admin": True,
+        "first_name": admin.first_name,
+        "last_name": admin.last_name,
+        "email": admin.email,
+    }
```

Comment:
Question: is there a specific reason to create an admin user here when `admin_user` and `admin_headers` are available test fixtures?

Reply:
I will modify it to use admin_user or admin_headers instead. I was following what test_get_all_users_as_admin on line 12 does for reference.